README.md

```diff
@@ -16,9 +16,10 @@
 ## Publishing to GitHub pages (`mozilla.github.io/irlpodcast`)
 
 1. Run the following build command: `JEKYLL_ENV=production jekyll build --config _config.yml,_config-build-dev.yml`
+1.1 If you need to see posts that are future dated (e.g. to verify an upcoming episode), add `--future` at the end of the above command.
```

Comment:
I think you intended for this to be a sub-bullet but right now it's a run-on sentence from `1.`

The use of `2.1` in the section above is not making a new separate bullet.